static/css/community.css

```
@@ -108,20 +128,29 @@ body.cid-community .community-section > p:not(.community-simple) {
   text-align: initial;
 }
 
+/* -------------------------------------
+ * Containers and blocks of sections - Introduction
+ ------------------------------------- */
 body.cid-community .community-section#introduction,
 body.cid-community .community-section#introduction > p {
   line-height: 1.75em;
   font-weight: 300;
   letter-spacing: 0.04em;
 }
 
+body.cid-community .community-section#introduction {
+  padding: 0 60px;
+  min-height: unset;
+}
+
+/* -------------------------------------
+ * Containers and blocks of sections - Conference gallery
+ ------------------------------------- */
 body.cid-community #gallery {
   display: flex;
-  max-width: 100vw;
+  flex-direction: row;
   gap: 0.75rem;
-  justify-content: center;
-  margin-left: auto;
-  margin-right: auto;
+  max-width: 100vw;
```

Let's stick with this
```suggestion
  max-width: 100%;
```